changeSdkVersion.ps1

```diff
@@ -24,13 +24,18 @@ if (-Not ($version -match "\d+\.\d+\.\d+")) {
 
 # update package.json 
 $packagesJson.version = $version
-$packagesJson | ConvertTo-Json | Out-File $packageJsonPath
+$packagesJson = $packagesJson | ConvertTo-Json 
+$packagesJson = $packagesJson -replace "\\u0026", "&"
```

Comment:
Json converter is encoding those characters. I need to revert them the original state.